docs/source/markdown/options/add-host.md

```diff
@@ -2,10 +2,28 @@
 ####>   podman build, create, farm build, pod create, run
 ####> If file is edited, make sure the changes
 ####> are applicable to all of those.
-#### **--add-host**=*host:ip*
+#### **--add-host**=*hostname[;hostname[;…]]*:*ip*|"host-gateway"
```

Comment:
Adding a space before and after pipe will fix the man-page-complainer, but I think that conveys the wrong impression. In fact no matter how I look at this, I can find no unambiguous way to express what I believe is the desired meaning. I think my only recommendation is to drop everything after pipe, and trust that the paragraph starting on line 16 will be clear enough.

Could I suggest changing the Unicode ellipsis to three ASCII dots?